#!/usr/bin/env python
PACKAGE = "fusion_pointclouds"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# name, param_type, level, description, default, min, max
#size_enum = gen.enum([gen.const("No" , int_t, 0,"No is 0."),
#                                                   gen.const("Yes", int_t, 1,"Yes is 1.")],
#                                                   "Select from the list")
#gen.add("PassThrough_Pc", int_t, 0,"Select from the list", 0, 0, 1,edit_method=size_enum)
gen.add("x_min", double_t, 0, "A double parameter",    -50, -50,   50)
gen.add("x_max", double_t, 0, "A double parameter",     50, -50,   50)
gen.add("y_min", double_t, 0, "A double parameter",    -50, -50,   50)
gen.add("y_max", double_t, 0, "A double parameter",     50, -50,   50)
gen.add("z_min", double_t, 0, "A double parameter",     -20, -20,   20)
gen.add("z_max", double_t, 0, "A double parameter",      20, -20,   20)

exit(gen.generate(PACKAGE, "fusion_pointclouds", "params_bounds"))
